_posts/2014-11-18-the-isnullorwhitespace-trap.html

```diff
@@ -116,7 +116,7 @@ <h3 id="5008ef2593b94c1582a52046d3c62ca4">
     Often, I see programmers implement their null checks like this:
   </p>
   <p>
-    <pre><span style="color:blue;">public</span>&nbsp;<span style="color:blue;">static</span>&nbsp;<span style="color:blue;">string</span>&nbsp;Canonicalize(<span style="color:blue;">string</span>&nbsp;searchTerm)
+  <pre><span style="color:blue;">public</span>&nbsp;<span style="color:blue;">static</span>&nbsp;<span style="color:blue;">string</span>&nbsp;Canonicalize(<span style="color:blue;">string</span>&nbsp;searchTerm)
```

Comment:
Github said this line was a conflict, but it didn't look like it to me and I wasn't working in this section of the doc, so idk